src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts

```diff
@@ -157,22 +159,39 @@ class FrameworkControllerTrainingService extends KubernetesTrainingService imple
         return Promise.resolve(trialJobOutputUrl);
     }
     
+    /**
+     * generate trial's command for frameworkcontroller
+     * expose port and execute injector.sh before user's command
+     * @param command 
+     */
+    private generateCommandScript(command: string): string {
+        let portScript = '';
+        if(!this.fcTrialConfig) {
+            throw new Error('frameworkcontroller trial config is not initialized');
+        }
+        for(let index in this.fcTrialConfig.taskRoles) {
```

Comment:
why use for...in..., rather than for...of... ?

Reply:
updated to 'for ... of ...'